spec/support/factory_girl.rb

```diff
@@ -7,4 +7,20 @@
     password 'password'
     password_confirmation 'password'
   end
+
+  factory :post do
+    title 'yeahhhhhhhooooo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.